src/main/java/parking/controllers/OwnerController.java

```diff
@@ -0,0 +1,39 @@
+package parking.controllers;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.PostMapping;
+import parking.entities.DateContainer;
+import parking.services.PricingService;
+
+import java.time.LocalDate;
+
+
+@Controller
+public class OwnerController {
+
+    private final PricingService pricingService;
+
+    @Autowired
+    public OwnerController(PricingService pricingService) {
+        this.pricingService = pricingService;
+    }
+
+    @GetMapping("/owner")
+    public String ownerGreeter(Model model) {
+        DateContainer dateContainer = new DateContainer();
+        dateContainer.setDateTime(LocalDate.now());
+        model.addAttribute("dateContainer", dateContainer);
+        return "owner";
+    }
+
+    @PostMapping("/getIncome")
+    public String getIncome(@ModelAttribute DateContainer dateContainer, Model model) {
```

Comment:
Czemu POST?

Reply:
Poczytałem sobie to:
https://martinfowler.com/articles/richardsonMaturityModel.html
potem poczytałem sobie to:
https://stackoverflow.com/questions/4226732/when-should-one-use-get-instead-of-post-in-a-web-application
I doszedłem do takich wniosków:
wolę użyć POST, żeby URL wyglądał tak:
```
http://localhost:8080/getIncome
```
zamiast (dla GET - bo rozumiem, że taki teoretycznie być powinien):
```
http://localhost:8080/getIncome?dateTime=2018-07-06
```
Jaka jest Twoja opinia?

Reply:
czemu dublujesz get?
czemu nie zrobić `GET /income`?

Reply:
No tak, już ustaliliśmy wcześniej (jak sobie poczytałem  https://restfulapi.net/resource-naming/), że nazewnictwo powinno być inne, np. bez czasowników, camel case i w ten sposób (opierając się o link) wołałbym `/GET income`. Tutaj już pozostałem, żeby wiadomo było do czego się odnoszę.